scala_modules/events/src/main/scala/io/dagster/events/EventPipeline.scala

```diff
@@ -33,34 +33,31 @@ package io.dagster.events
   * We currently use AWS 1.7.4 and hadoop-aws 2.7.1 as these are known to be compatible and work with Spark 2.4.0.
   */
 
-import org.apache.spark.sql.{Dataset, SparkSession}
+import java.io.File
+
+import org.apache.spark.sql.Dataset
 import java.util.Date
 
-import com.amazonaws.auth.BasicAWSCredentials
 import com.amazonaws.services.s3._
 import model._
 
 import scala.reflect.internal.FatalError
 import scala.collection.JavaConversions._
 import scala.io.Source
-import models.{Event, LocalStorageBackend, S3StorageBackend, StorageBackend}
+import models._
```

Comment:
what is the scala best practice in general for dumping a bunch of symbols into global scope? I get scared when is see stuff like:

```
import model._
```

followed by 

```
import models._
```

I suppose a good IDE goes along way in helping here

Reply:
argh, this was me trusting IntelliJ too much, thanks for catching. This is bad practice.

I usually try to follow the Twitter Effective Scala style guide: http://twitter.github.io/effectivescala/#Formatting-Imports 

At some point I'll take care of this w/ #979